src/careamics/config/tile_information.py

```diff
@@ -17,26 +17,26 @@ class TileInformation(BaseModel):
 
     model_config = ConfigDict(validate_default=True)
 
-    array_shape: Tuple[int, ...]
+    array_shape: tuple[int, ...]
     tiled: bool = False
     last_tile: bool = False
-    overlap_crop_coords: Optional[Tuple[Tuple[int, ...], ...]] = Field(default=None)
-    stitch_coords: Optional[Tuple[Tuple[int, ...], ...]] = Field(default=None)
+    overlap_crop_coords: Optional[tuple[tuple[int, ...], ...]] = Field(default=None)
+    stitch_coords: Optional[tuple[tuple[int, ...], ...]] = Field(default=None)
 
     @field_validator("array_shape")
     @classmethod
-    def no_singleton_dimensions(cls, v: Tuple[int, ...]):
+    def no_singleton_dimensions(cls, v: tuple[int, ...]):
         """
         Check that the array shape does not have any singleton dimensions.
 
         Parameters
         ----------
-        v : Tuple[int, ...]
+        v : tuple[int, ...]
```

Comment:
How do you feel about:
```python
"""
v : tuple of integers
"""
```
or 
```python
"""
v : tuple of (int, ...)
"""
```
Similar to the [Pandas docstring guide](https://pandas.pydata.org/docs/development/contributing_docstring.html#parameter-types) although they don't have an example for unknown length tuples.

Reply:
I like the first one, makes it quite readable!